src/components/Articles/Articles.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState,useEffect } from "react"
+import {getArticles} from '../../utils/api'
+import './Articles.css'
+
+
+
+
+const Articles=()=>{
+  const [allArticles,setAllArticles]=useState([])
+  const [isLoading,setIsLoading]=useState(true)
+  useEffect(() => {
+    setIsLoading(true)
+    getArticles().then((articles) => {
+      setAllArticles(articles);
+      setIsLoading(false)
+    });
+  }, []);
+
+  if (isLoading) return <p>LOADING .....</p>
+
+
+return (
+    <div id="articles-container">
+      {allArticles.map((article, index) => (
+        <div key={index} className="article-card">
+          <img src={article.article_img_url} alt={article.title} />
+          <h3>{article.title}</h3>
+          <p>{article.author}</p>
+          <span>Votes: {article.votes}</span>
```

Comment:
We also want to see when the articles were created in a human readable format as they should already be sorted by date when they come back from the api, as well as the comment count too